marklogic-data-hub-central/ui/e2e/cypress/support/pages/browse.tsx

```diff
@@ -338,6 +338,10 @@ class BrowsePage {
     return cy.findByPlaceholderText("Search").clear();
   }
 
+  getSearchTextElement() {
+    return cy.findByPlaceholderText("Search");
+  }
```

Comment:
![image](https://github.com/marklogic/marklogic-data-hub/assets/126084579/f83e7d2d-1f82-4963-ac06-34290b6e42bb)

![image](https://github.com/marklogic/marklogic-data-hub/assets/126084579/81a45da0-d50c-4bbc-ae45-e43679c7f3e9)


The getSearchText getter should be replaced in all files that is being used, it is not a good idea to duplicate getters. The getSearchTextElement is bassically the same that getSearchText, so everything should be change to getSearchTextElement and then delete getSearchText() getter.

Reply:
The issue with the original function has added a clear function at the end, if I change that function in all files maybe I can cause a lot of e2e failures maybe we can create a ticket to refactor all of that ?